symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/MessageService.java

```diff
@@ -241,6 +249,11 @@ private V4MessageBlastResponse doSendBlast(List<String> streamIds, Message messa
   }
 
   private Map<String, Object> getForm(Message message) {
+    try {
+      this.messageMLValidator.dataExceededValidate(message);
+    } catch (InvalidInputException | IOException | ProcessingException e) {
```

Comment:
I would prefer to have this try...catch statement directly put in `dataExceededValidate(message)` method to have all the logic contained in one place.